content/try-opensds/installation-using-ansible.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Installation Using ansible
+description: "This is a  guide for users and new contributors to install Local Cluster using ansible."
+weight: 30
+---
+This document describes how to install SODA projects local cluster, including Hotpot (API, Controller, Dock), Gelato, Telemetry, Orchestration, and Dashboard components.
+
+`Hereafter, Hotpot refers to API, Controller and Dock projects`
+
+### Pre-config (Ubuntu 16.04)
+All the installation work is tested on `Ubuntu 16.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+
+<br /> Install following packages:
```

Comment:
nit: use #### for subheadings  

### Pre-config (Ubuntu 16.04)
....
#### Install following packages
...
#### Install Docker
....
#### Install docker-compose